workspaceitem-data-upload.md

```diff
@@ -5,9 +5,11 @@ The section data represent the data about the user uploaded files
 
 ```json
 {
+    "primary": "00001abf-b2e0-477a-99de-104db7cb6469",
 	"files": [ 
   	 	{
-  	 		metadata: {
+            "id":  "00001abf-b2e0-477a-99de-104db7cb6469",
```

Comment:
In the responses, when I test the backend, this is called `uuid` instead of `id`.  So, I think this example needs to say `uuid`.